### PR TITLE
added logic for saving criteria

### PR DESCRIPTION
**Pull request by HussainKhanSherwani**
# Pull Request Template

Before submitting this PR, please make sure you have completed the following checklist. This helps ensure that our codebase remains high quality and that our features work well across all supported devices and themes.

### Code Review
- [ ] I have carefully reviewed my code and believe it is final from my side.

### Functional Testing
- [ ] I have thoroughly tested the functionality of my changes.
- [ ] I am confident that my changes do not negatively impact existing features.

### Testing Documentation
Please provide a brief description of the tests you have performed to verify your changes. Include any relevant scenarios or edge cases.

- Example test 1: Tested feature X by uploading the image
- Example test 2: Tested feature X by doing these things.

# Demo Video
Please provide a demo video of your changes. This helps reviewers understand the changes and verify the functionality.

You can record the video using tools like Loom(https://www.loom.com/), Clip by ClickUp(https://clickup.com/features/clips), Sendspark(https://www.sendspark.com/), etc.

Include the link to the video below and make sure the video covers
- [ ] Explain the task/bug you are solving
- [ ] Show the changes in the codebase
- [ ] Show the changes you tested to make sure it works

### CODE REVIEW

File: crowd-fund-analysis/cf_analysis_agent/api/public_equity_api.py [hunk 1]
```diff
@@ -1,20 +1,34 @@
...
-        subIndustry = equity_details.get("subIndustry", "").lower().replace(" ", "-")
+        equity_details:EquityDeatils = request.json  # Get the JSON request body
+        sector = equity_details.get("sector", "").get("name","").lower().replace(" ", "-")
+        industryGroup = equity_details.get("industryGroup", "").get("name","").lower().replace(" ", "-")
+        industry = equity_details.get("industry", "").get("name","").lower().replace(" ", "-")
+        subIndustry = equity_details.get("subIndustry", "").get("name","").lower().replace(" ", "-")
         s3_key = f"{sector}/{industryGroup}/{industry}/{subIndustry}/agent-status.json"
         print("Received Equity Details:", equity_details)
         upload_equity_project_to_s3(json.dumps(equity_details), s3_key , content_type="application/json")
+        industry_group_criteria:StructuredIndustryGroupCriteriaResponse  = get_industry_group_criteria(sector, industryGroup,industry,subIndustry)
```
[the remark this answers is not in the document]
Comment (HussainKhanSherwani) [reply crowd-fund-analysis/cf_analysis_agent/api/public_equity_api.py:20]: sure

File: crowd-fund-analysis/cf_analysis_agent/api/public_equity_api.py [hunk 2]
```diff
@@ -1,20 +1,41 @@
...
 
 @public_equity_api.route("/submit", methods=["POST"])
 def submit_equity_project():
     try:
-        equity_details = request.json  # Get the JSON request body
-        sector = equity_details.get("sector", "").lower().replace(" ", "-")
-        industryGroup = equity_details.get("industryGroup", "").lower().replace(" ", "-")
-        industry = equity_details.get("industry", "").lower().replace(" ", "-")
-        subIndustry = equity_details.get("subIndustry", "").lower().replace(" ", "-")
+        equity_details:EquityDeatils = request.json  # Get the JSON request body
```
Comment (RobinNagpal) [inline crowd-fund-analysis/cf_analysis_agent/api/public_equity_api.py:12]: We should write a proper slugify method and use it
